build.gradle

```diff
@@ -51,6 +51,20 @@ tasks.named(sourceSets.main.processResourcesTaskName, Copy) {
     from(actionsTemplateZip)
 }
 
+tasks.named('jar', Jar) {
+    final ByteArrayOutputStream bos = new ByteArrayOutputStream()
+    project.exec {
+        commandLine('git', 'rev-parse', 'HEAD')
+        standardOutput(bos)
+    }.rethrowFailure().assertNormalExitValue()
```

Comment:
Should this be wrapped in a `try`-`catch`, like in the `GenerateChangelog` task class down below, to add an informative message about checking if `git` is on the PATH?